test/unit/api/wide/load/arithmetic/load.hpp

```diff
@@ -131,8 +131,8 @@ TTS_CASE_TPL("Check conditional load from pointer for wide", EVE_TYPE)
     auto i1 = eve::ignore_first(EVE_CARDINAL/4);
     auto kf = eve::keep_first(EVE_CARDINAL/4);
     auto kl = eve::keep_last(EVE_CARDINAL/4);
-    auto kb = eve::keep_between ( std::min(0L,EVE_CARDINAL/3)
-                                , std::max(0L,(EVE_CARDINAL*2)/3)
+    auto kb = eve::keep_between ( std::min<std::ptrdiff_t>(0L,EVE_CARDINAL/3)
+                                , std::max<std::ptrdiff_t>(0L,(EVE_CARDINAL*2)/3)
```

Comment:
Specifing template parameters for standard template parameters is no the supported usecase as far as I understand. You can cast though

Reply:
OK will cast